core/crypto/vrf/vrf_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vrf
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestUniqueID(t *testing.T) {
```

Comment:
Hmm...  I don't think so.  (Happy to be convinced otherwise).
Do we want to fix the UniqueID format in the test? 

As is, the tests exercise a possible attack scenarios and make sure that unique values are being produced from unique inputs.